docker-mcp-gateway/mcp-config.sh

```
@@ -0,0 +1,132 @@
+      show_help
+      exit 1
+      ;;
+  esac
+done
+
+# 設定ファイル生成
+CONFIG_FILE="docker-mcp-config.json"
+echo "MCP設定ファイルを生成します: $CONFIG_FILE"
+
+# 基本コマンド
+CMD_ARGS=("mcp", "gateway", "run")
```

Shellcheck (0.11.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)